workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSSendSystem.cs

```diff
@@ -55,16 +68,23 @@ protected override void OnUpdate()
                 return;
             }
 
-            var connection = worker.Connection;
-
-            foreach (var componentTypeIndex in registeredReplicators)
+            foreach (var replicator in specificComponentReplicators)
             {
-                view.TranslationUnits[componentTypeIndex].ExecuteReplication(connection);
+                replicator.Execute(worker.Connection);
             }
+        }
+
+        private struct SpecificComponentReplicator
```

Comment:
Why "specific"

Reply:
Unless there is a more general one we could just call it ComponentReplicator though

Reply:
I mean, I guess 😉